contracts/FairLaunchClaimer.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "./AuctionSuperApp.sol";
+import "./GeoWebParcel.sol";
+import "./ERC721License.sol";
+import "./interfaces/IClaimer.sol";
+
+contract FairLaunchClaimer is Pausable, AccessControl, IClaimer {
+    bytes32 public constant PAUSE_ROLE = keccak256("PAUSE_ROLE");
+    bytes32 public constant CLAIM_ROLE = keccak256("CLAIM_ROLE");
+
+    /// @notice License
+    ERC721License public license;
+
+    /// @notice Parcel
+    GeoWebParcel public parcel;
+
+    /// @notice Auction
+    AuctionSuperApp public auctionApp;
```

Comment:
Is this actually used anywhere?

Reply:
not in this contract, it will be used in the reclaimer, i can remove it here